src/dbos-runtime/config.ts

```diff
@@ -166,8 +165,6 @@ export function parseConfigFile(cliOptions?: DBOSCLIStartOptions, useProxy: bool
     throw new DBOSInitializationError(`dbos-config.yaml failed schema validation. ${errorMessages}`);
   }
 
-  setApplicationVersion(configFile.version);
```

Comment:
So we also need to update dbos-config.schema.json to remove the `version` field in this PR. Otherwise, it could be confusing to people.